src/classes/RD2_OpportunityService.cls

```diff
@@ -118,34 +118,28 @@ public with sharing class RD2_OpportunityService {
     public RD2_OpportunityService createOpportunities(List<npe03__Recurring_Donation__c> rds) {
         List<Opportunity> oppsToCreate = new List<Opportunity>();
         Map<Id, List<RD2_ScheduleService.Installment>> installmentsByRDId = new Map<Id, List<RD2_ScheduleService.Installment>>();
-        Integer numberOfInstallments = 1;
 
         for (npe03__Recurring_Donation__c rd : rds) {
-            List<RD2_ScheduleService.Installment> installments = scheduleService.getVisualizedInstallments(
-                currentDate, numberOfInstallments, rd.RecurringDonationSchedules__r
-            );
-            installmentsByRDId.put(rd.Id, installments);
+            RD2_OpportunityMatcher.Record nextDonationDateRecord = scheduleService.getNextDonationDateRecord(rd, rd.RecurringDonationSchedules__r);
```

Comment:
Can we revert the Opp service `createOpportuniteis()`  to be the same as in the parent Opp matcher branch? Changes should not include the next donation date code.